nemo/collections/nlp/parts/nlp_overrides.py

```diff
@@ -499,6 +504,16 @@ def _fix_param_groups(
         model_has_expert_param = any(param.get('is_expert', False) for param in model_param_groups)
         checkpoint_has_expert_param = any(param.get('is_expert', False) for param in checkpoint_param_groups)
 
+        patch_pipeline_index = None
+        if patch_separate_lm_head_stage:
+            patch_pipeline_index = len(model_param_groups)
+            model_param_groups.insert(patch_pipeline_index, {'params': LocalNonpersitentObject([0])})
```

Comment:
Curious - any difference between `[]` and `[0]` when creating `LocalNonpersitentObject`?